test/cli/createSecret.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import json
+import argparse
+import base64
+import time
+import random
+import hmac
+from hashlib import sha256
+from collections import OrderedDict
+import urllib.parse
+
+import _utils_
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--url', type=str, help='the address of the ehsm_kms_server', required=True)
+    parser.add_argument('--secretData', type=str, help='the value of the secret to be created', required=True)
+    parser.add_argument('--secretName', type=str, help='the name of the secret', required=True)
+    parser.add_argument('--encryptionKeyId', type=str, help='The ID of the CMK that is used to encrypt the secret value')
+    parser.add_argument('--description', type=str, help='the description of the secret')
+    parser.add_argument('--rotationInterval', type=str, help='Automatic rotation interval')
+    args = parser.parse_args()
+
+    base_url = args.url + "/ehsm?Action="
+    print(base_url)
+    return base_url, args.secretData, args.secretName, args.encryptionKeyId, args.description, args.rotationInterval
+
+def createSecret(base_url, secretData, secretName, encryptionKeyId, description, rotationInterval):
+    print('create secret')
+    
+    payload = OrderedDict()
+    payload["description"] = description
+    if encryptionKeyId != None:
+        payload["encryptionKeyId"] = encryptionKeyId
```

Comment:
so how about the comman line looks like?

does the user need to provide the encryptionkeyid as None? can you show me the details about the cmdline?